TLM/TLM/UI/SubTools/PrioritySignsTool.cs

```diff
@@ -51,95 +51,26 @@ public PrioritySignsTool(TrafficManagerTool mainTool)
                     RefreshMassEditOverlay();
                     return;
                 }
-            // TODO uncomment as part of issue #541
-            //} else if (ctrlDown) {
-            //
-            //    //PriorityRoad.FixJunction(HoveredNodeId);
-            //    RefreshMassEditOverlay();
-            //    return;
+                // TODO uncomment as part of issue #541
+                //} else if (ctrlDown) {
+                //
+                //    //PriorityRoad.FixJunction(HoveredNodeId);
+                //    RefreshMassEditOverlay();
+                //    return;
             }
             if (shiftDown) {
-                var primaryPrioType = PriorityType.None;
```

Comment:
moved this code into a method so that I can call it from my road selection panel.